Migration API and migration:import command (#1315)
ezsystems/developer-documentation: docs/guide/data_migration.md

The converter handles Kaliop types that are different from Ibexa types.

``` bash
php bin/console ibexa:migrations:kaliop:convert --input=kaliop_format.yaml --output=ibexa_format.yaml
```

You can also convert multiple files using `ibexa:migrations:kaliop:bulk-convert`:

``` bash
php bin/console ibexa:migrations:kaliop:bulk-convert --recursive --input-directory=kaliop_files --output-directory=ibexa_files
```

If you do not specify the output directory, the command overwrites the input files.

## Adding migration files

> Use the `ibexa:migrations:import` command to add files to the defined folder for migrations

Reviewer: ```suggestion
Use the `ibexa:migrations:import` command to add files to the directory defined in configuration
```
